jax/_src/test_util.py

```diff
@@ -2036,3 +2044,47 @@ def worker(ctx, s, e, r, v):
       return worker(ctx, scale, exact, reference, value)
     else:
       assert 0  # unreachable
+
+# Hypothesis testing support
+def setup_hypothesis(max_examples=30) -> None:
```

Comment:
@gnecula nit: this should be `set_up_hypothesis` because "setup" is a noun while "set up" is a verb phrase :P

Reply:
Good point, I will include this cleanup in a future PR